requirements/example.txt

```diff
@@ -4,7 +4,7 @@ Cartopy>=0.18
 shapely
 # Required for Cartopy
 osmnx>=0.16
-pygeos==0.8
+geopandas>=0.8.2
```

Comment:
Now that's geopandas 0.82 has been released, it will be installed by ``momepy>=0.4``.  So let's remove ``geopandas>=0.8.2``.